Code/Framework/AzCore/AzCore/Instance/InstancePool.h

```diff
@@ -24,19 +24,28 @@ namespace AZ
 {
     class InstancePoolBase
     {
+    public:
+        virtual ~InstancePoolBase()
+        {
+        }
+        AZ_RTTI(InstancePoolBase, "{B32CEB58-7744-4275-92BE-BFC850E3C4CD}");
```

Comment:
-it's common to put `AZ_RTTI` first.
-Also for empty destructors it's preferred to use `virtual ~InstancePool() = default;`

````
        AZ_RTTI(InstancePoolBase, "{B32CEB58-7744-4275-92BE-BFC850E3C4CD}");

        virtual ~InstancePoolBase() = default;
````

Reply:
sure, changed now.